src/components/movieListItem/MovieListItem.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import MovieListItemInfo from '../movieListItemInfo';
+import MovieListItemMain from '../movieListItemMain/MovieListItemMain';
+
+const MovieListItem = ({ movie, genres }) => {
+  const {
+    id, title, rating, posterPath, genreIds, overview, isMain,
+  } = movie;
+  const movieCover = {
+    background: `url(https://image.tmdb.org/t/p/w300/${posterPath}) no-repeat center top / cover`,
+  };
+  let genreStr = '';
+  if (genres) {
+    genreStr = genreIds.map((itemId) => genres.find((g) => g.id === itemId).name).join(', ');
+  }
```

Comment:
IMHO, I would create a function which gets as params `genreIds` and `genres` and returns `genreStr` instaed of defining veriable `genreStr`.
It's just an option. Choose what you like more.